### ISSUE TITLE
Should Google Maps be able to find location addresses in Incognito?

### ISSUE THREAD

**Issue by adambard1** [streetlives/yourpeer.nyc#192]
I am not sure how to test the Get Directions result for somone who is not sharing their location.
Even if I am in incognito, I appear to be sharing my location with Google Maps, so can't see what the rsult would look like.

**Comment by jbeard4** [comment]
Yes, this is expected. Incognito mode does not block the user' location. it just prevents the browser form persisting cookies that were set while the user was browsing in incognito mode.
